### DISCUSSION TITLE
Grafana Status Map Panel not working, shows "Panel plugin not found: flant-statusmap-panel"

### DISCUSSION THREAD

**Discussion by danielaporcescu** [flant/grafana-statusmap#231] [Q&A]
**What happened**:
Grafana Status Map Panel was working fine before, but now shows "Panel plugin not found: flant-statusmap-panel". The issue was spotted about 2 weeks ago and it never recovered (worked again).

**What you expected to happen**:
To be working.

**How to reproduce it (as minimally and precisely as possible)**:
Create an Azure App Service and install the plugin by going into Configuration, and adding a new plugin like: GF_INSTALL_PLUGINS="flant-statusmap-panel".

**Environment**:
- Grafana version: v7.5.4 (615c153b3a)
- flant-statusmap-panel version: no idea
- Data source type & version: database for PostgreSQL server
- OS Grafana is installed on: Linux (runs as a docker image in the Azure App Service)
- User OS & Browser: Windows, Chrome
- Grafana plugins: flant-statusmap-panel"
- Others:

**Anything else we should know?**:
Nothing was changed in the application for the plugin not to work. The application was restarted to see if the issue still persists, and it does.

**Comment by diafour** [comment]
Hello! Is this issue still persist?

I think the problem is a misconfiguration, not a bug. It helps if you get a log from the Grafana container and attach it here. There should be lines about loading plugins. Successful loading of the statusmap looks like this:

```
DBUG[05-24|16:57:35] Found plugin                             logger=plugins id=flant-statusmap-panel signature=unsigned hasRoot=false
DBUG[05-24|16:57:35] Non-valid plugin Signature               logger=plugins pluginID=flant-statusmap-panel pluginDir=/flant-statusmap-panel/dist state=unsigned
DBUG[05-24|16:57:35] Attempting to add plugin                 logger=plugins id=flant-statusmap-panel
INFO[05-24|16:57:35] Registering plugin                       logger=plugins id=flant-statusmap-panel
DBUG[05-24|16:57:35] Successfully added plugin                logger=plugins id=flant-statusmap-panel
```

**Comment by diafour** [comment]
Also, I have no experience with Azure App Service. Can you add more details or attach a screenshot of this action:

> install the plugin by going into Configuration, and adding a new plugin like: GF_INSTALL_PLUGINS="flant-statusmap-panel".

**Comment by aol-nnov** [comment]
I'm experiencing it as well.

<img width="276" alt="image" src="https://user-images.githubusercontent.com/43190/121634279-eddf5200-ca8c-11eb-9834-4aab7bb2f93a.png">

UPD: ahem... I'd better be updating grafana/plugin before chimning in...

It's working fine now. Grafana v8.0.1 (8849243d27) and latest version of statusmap plugin.

Sorry for the noise.
